css/styles.css

```diff
@@ -0,0 +1,368 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap");
+
+* {
+  margin: 0;
+  padding: 0%;
+  box-sizing: border-box;
+}
+
+body {
+  position: relative;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+}
```

Comment:
- [ ] Kindly make sure your home page, about and card sections look as they look in Figma design. Add margins (top, bottom, left, right) defined in the design as defined in the design. You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma]
## Home page 
![image](https://user-images.githubusercontent.com/95297251/185767157-b859217c-f83e-4022-ac56-afb732484844.png)
## card section
![image](https://user-images.githubusercontent.com/95297251/185767204-9d0edd49-08e2-478c-ae2d-e2b99bde424c.png)
## about section
![image](https://user-images.githubusercontent.com/95297251/185767134-1b504d4e-14c4-433d-b6ee-be08bf2237fa.png)